presentation/src/main/java/com/kk/presentation/player/GamingRoomViewModel.kt

```diff
@@ -0,0 +1,40 @@
+package com.kk.presentation.player
+
+import com.kk.presentation.baseMVI.BaseViewModel
+
+class GamingRoomViewModel :
+    BaseViewModel<GamingRoomContract.Event, GamingRoomContract.State, GamingRoomContract.Effect>() {
+
+    override fun createInitialState(): GamingRoomContract.State {
+        return GamingRoomContract.State(
+            GamingRoomContract.GamingRoomState.Idle
+        )
+    }
+
+    override fun handleEvent(event: GamingRoomContract.Event) {
+        when (event) {
+            is GamingRoomContract.Event.OnQuestionButtonClicked -> {
+                onQuestionButtonClcik()
+            }
+
+            is GamingRoomContract.Event.OnQuestionSkipButtonClicked -> {
+                skipQuestion()
+            }
+            is GamingRoomContract.Event.SendAnswerButtonClicked -> {
+                sendAnswer()
+            }
+        }
+    }
+
+    private fun onQuestionButtonClcik() {
```

Comment:
Change the name of this function, the event name is OnQuestinButtonClicked, but you need to name te function diferent because is going to do another thing ,

Reply:
fixed in new push